presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableWriterNode.java

```diff
@@ -246,10 +251,10 @@ public SchemaTableName getSchemaTableName()
         }
 
         @Override
-        public String toString()
-        {
-            return handle.toString();
-        }
+        public String toString() { return handle.toString(); }
+
+        @Override
+        public String toPlanString() { return handle.toPlanString(); }
```

Comment:
Instead of introducing new interface, let's just implement this method directly: 

`return handle.getConnectorId() + ":" + handle.getConnectorHandle();`

Reply:
Good idea, I prefer this approach.